CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
-# wai-log-0.2 (not released)
+# wai-log-0.3 (not released)
+
+* Rename `mkApplicationLogger` into `mkLogMiddleware`
+* Generate ID for each request to correlate log messages
+* Remove optional `logSendingResponse`, which means always logging `Sending response`
+* Log `method` and `url` in `Request complete` too
+* Log `response_body` on 4xx and 5xx
```

Comment:
Not done directly in this library anymore.

Reply:
done.